src/rgw/rgw_lc.cc

```diff
@@ -2002,27 +2002,6 @@ int RGWLC::process_bucket(int index, int max_lock_secs, LCWorker* worker,
   std::unique_lock<rgw::sal::LCSerializer> lock(
     *(serializer.get()), std::adopt_lock);
 
-  if (! (cct->_conf->rgw_lc_lock_max_time == 9969)) {
```

Comment:
i'm confused, these blocks run when rgw_lc_lock_max_time does *not* have this special debug value

but if the intent of this code is to prevent a bucket from being processed by two gateways at the same time, and the `LCSerializer` is already preventing that from happening, maybe we don't need it?

Reply:
no, sorry, you're right, this isn't what I intended to do

Reply:
in the RGWLC::process(...) case, though I think it is correct for RGWLC::process_bucket(...)

Reply:
but, the lcserializer isn't doing that job;  it's only protecting the consistency of the entry queue--the lcserializer is dropped across bucket_lc_process

Reply:
in any case, we only want to drop the condition, not the logic in either of the blocks (as it fetches the lc entry)